test/field/RungeKutta.test.cu

```diff
@@ -14,6 +14,7 @@
 #include "base/Range.hh"
 #include "base/Types.hh"
 #include "base/Units.hh"
+#include "field/DormandPrinceStepper.hh"
```

Comment:
Since you're making this a test of both steppers, perhaps rename these files to `Steppers.test.xx`?

Reply:
Done.